### ISSUE TITLE
Feautre Extraction Input Audio???

### ISSUE THREAD

**Issue by vi-villageai** [wentaozhu/speechnas#3]
I don't see Input model which forward model to predict embedings? 
No see function "extract 80-dimensional MFCCs"

**Comment by wentaozhu** [comment]
Thank you so much for the interest!

You can download the 30-dimensional MFCCs without augmentation from the BaiduNetDisk Link: https://pan.baidu.com/s/1c1lAkAgjvj4k2zIYCm__qA  with password: ad44

and set the path https://github.com/wentaozhu/speechnas/blob/main/dataloader/voxceleb.py#L427 as speech_data_no_aug/csv_file

You can run inference and get the metric from https://github.com/wentaozhu/speechnas/tree/main/metric/metric_eer

I am uploading the augmented 80-dimensional MFCCs.

I will upload the Kaldi environment and do a comprehensive test shortly. Thank you so much!

**Comment by vi-villageai** [comment]
Thanks WentaoZhu! I hope to see your test shortly soon. Because I want to apply your model for training new model with other dataset.

**Comment by wentaozhu** [comment]
Both 30-dimensional MFCCs without augmentation and 80-dimensional MFCCs with 6 times augmentation can be downloaded from link: https://pan.baidu.com/s/1g4OaR55YXH-oC35XJ9kWCg  password: so72

For augmented dataset, you can unzip by
`zip data_six_aug_combined_sub.zip -s=0 --out data_six_aug_combined.zip;`
`unzip -d ./ data_six_aug_combined.zip`

You may try to check the error rate with pretrained model in https://github.com/wentaozhu/speechnas/tree/main/checkpoint with network configuration in https://github.com/wentaozhu/speechnas/tree/main/network and metric calculation in https://github.com/wentaozhu/speechnas/tree/main/metric/metric_eer

I will upload the Kaldi and calculate the EER for all the networks shortly.

Thank you so much for the interest!
